### PR TITLE
Fixed the animation

### PR DESCRIPTION
**Pull request by evandroabukamel**
Round resize function startY value. 
Organize country-table-body creation. 
Convert NaN values to 0. 
Add some comments to functions declaration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -258,34 +258,52 @@ <h1 class="font-weight-bold pb-3">SOURCES</h1>
         var start = moment("2018-01-01");
         var percentageBudgetLeft = 0;
 
+        /**
+         * Format a number to string format, with commas to separate groups of thousands.
+         */
         function numberFormater(numberToFormat) {
```
Comment (moredatapls) [inline index.html]: I just noticed that there's a typo, should be called `numberFormatter()` with two T. Would you mind fixing this?

**Comment by moredatapls** [comment]
Beautiful, I love documented code! Thank you! Will merge it once the typo is gone

**Comment by evandroabukamel** [comment]
Sure, I will fix it, I didn't notice too.

**Comment by evandroabukamel** [comment]
@moredatapls , I noticed that the menu doesn't link to SOURCES section. Can I add it?

**Comment by moredatapls** [comment]
Absolutely!

**Comment by evandroabukamel** [comment]
Done.

**Comment by moredatapls** [comment]
Resolves #4

**Pull request merged**